packages/@aws-cdk/aws-cloudwatch/README.md

```diff
@@ -85,6 +85,18 @@ const problemPercentage = new MathExpression({
 })
 ```
 
+Math expressions also support pre-defined CloudWatch math functions like
+[anomaly detection](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Anomaly_Detection.html):
+
+```ts
+const invocationAnomalies = new MathExpression({
```

Comment:
@NetaNir This is how I envisioned the new feature would work with `MathExpression`. This new features appears to be considerably more work than I anticipated when I first picked this up. If it's a priority for you and / or your colleagues, I'm happy to hand this over, otherwise with your advice I can proceed. Let me know your thoughts before I make any further developments.

Reply:
The latest version of the API looks ok. Since this feature is non trivial. please add an integration test so we can verify the deployment succeed.